addons/website_quote/views/website_quote_templates.xml

```diff
@@ -321,6 +321,7 @@
                                     <t t-set="icon_class" t-value="'fa-lock'"/>
                                     <t t-set="form_action" t-value="'/quote/' + str(quotation.id) + '/transaction/token'"/>
                                     <t t-set="prepare_tx_url" t-value="'/quote/' + str(quotation.id) + '/transaction/'"/>
+                                    <t t-set="access_token" t-value="quotation.access_token"/>
```

Comment:
Commit ('fix redirection from'): I would just add an explanation that access token is taken into account and propagated in the payment form and that's why we have to add this line.